### ISSUE TITLE
Super Terceira Idade

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments-new#152]
### Post de Referência:

https://www.vish.blog.br/imagens/super-terceira-idade/

**Comment by brunoaugusto** [comment]
A cara do Hulk "ODEIO MÉDICO!" :rofl:

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/super-terceira-idade/#comment_1187604821>):

> A cara do Hulk "ODEIO MÉDICO!" 🤣

Isso se é que aquela placa diz "médico" ou algo do tipo :thinking:

**Comment by najarin-the-loremaster** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/super-terceira-idade/#comment_1187610699>):

> Isso se é que aquela placa diz "médico" ou algo do tipo 🤔

Está escrito "Oculista". Oftalmologista, que seria o mais apropriado, é uma palavra bem maior em russo (офтальмолог)

**Comment by maryon-telvanni** [comment]
> [Najarin, the Loremaster](</perfil/?user=najarin-the-loremaster>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/super-terceira-idade/#comment_1187633177>):

> Está escrito "Oculista". Oftalmologista, que seria o mais apropriado, é uma palavra bem maior em russo (офтальмолог)

Você lê/fala russo, Najarin?

**Comment by najarin-the-loremaster** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [segunda-feira, 18 de julho de 2022](<http://localhost:1313/imagens/super-terceira-idade/#comment_1187653374>):

> Você lê/fala russo, Najarin?

Não, sou fluente em Google Translate ou, nesse caso, [Yandex Translate](<https://translate.yandex.com/ocr>) que traduziu o texto a partir de um pedacinho da imagem. Mas traduzir cada caractere do [Alfabeto Cirílico](<https://pt.wikipedia.org/wiki/Alfabeto_cir%C3%ADlico>) também daria certo
